prov/efa/src/efa_msg.c

```diff
@@ -391,7 +392,7 @@ static ssize_t efa_post_send(struct efa_ep *ep, const struct fi_msg *msg, uint64
 	if (flags & FI_MORE)
 		return 0;
 
-	ret = efa_post_flush(ep, &bad_wr);
+	ret = efa_post_flush(ep, &bad_wr, true);
```

Comment:
add `true /* free */` to indicate is meaning

Reply:
done